pegasus/sites.v3/code.org/views/mobile_footer_responsive.haml

```diff
@@ -1,11 +1,48 @@
--# an utra-simple mobile footer
+%link{:href=>'/css/mobile-footer.css', :rel=>'stylesheet'}
+%script{src: '/js/mobile-footer.js'}
 
 %footer#pagefooter
-  .content{:style=>'width:100%'}
-    .left{:style=>'float:left'}
-      - if request.language == "en"
-        %a.whitefooterlink{:href=>"/privacy", :style=>"color: #ffa400;"} Privacy Policy
-      %br
-      !="&copy; Code.org, #{Time.now.year}"
-    .right{:style=>'float:right'}
+  #footer-links
+    %ul.hide-responsive-footer-menu
+      %li
+        %a{:href=>'/privacy', :style=>'color: #ffa400;', :target=>'_blank'} Privacy Policy
+      %li
```

Comment:
This seems like a potential candidate for an array and some code that iterates through that array.